system-monitor@paradoxxx.zero.gmail.com/extension.js

```diff
@@ -1082,7 +1082,11 @@ const Battery = class SystemMonitor_Battery extends ElementBase {
         this.icon_hidden = false;
         this.percentage = 0;
         this.timeString = '-- ';
-        this._proxy = StatusArea.aggregateMenu._power._proxy;
+        if (shell_Version => '43') {
```

Comment:
This is actually just defining an arrow function, which will always evaluate to true. I believe you want `shell_Version >= '43'`)

Reply:
Yes you are right, i maked a typo, i have just corrected it.
Thanks

Reply:
:+1: